roboteam_ai/src/control/ballHandling/RotateAroundRobot.h

```diff
@@ -0,0 +1,32 @@
+//
```

Comment:
Name this rotateWithBall or so, all of AI was confused about this one.

Reply:
fixed